packages/tutorials/election-dapp/front-end/src/utils.ts

```diff
@@ -0,0 +1,52 @@
+import { poll } from '@kadena/chainweb-node-client';
+
+const delay = (millis: number) => new Promise<void>(resolve => {
+  setTimeout(() => resolve(), millis)
+})
+
+/**
+ * Poll for transaction mined on the blockchain
+ *
+ * @param requestKeys - Array of request keys to poll
+ * @param apiHost - API host
+ * @return
+ */
+export async function pollTransactions(
```

Comment:
If I remember correctly, kadena.js now supports this from within the library

Reply:
Indeed, I've updated the code to use the one from `@kadena/client`